sdk/src/main/res/values/attrs.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <attr name="omiseButtonStyle" format="reference" />
+    <attr name="omiseEditTextStyle" format="reference" />
+    <attr name="omiseEditTextLabelStyle" format="reference" />
```

Comment:
Do we have an alternative for this approach? Is it ok to use the existed style/attribute?

Reply:
1. For setting the style via styles.xml seems to be only one way that think out for now. 
2. I think it betters existed style from the API cause the API it too general and I can't guarantee it'll not affect to another component.